patches/@storybook+builder-webpack5+6.5.14.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/node_modules/@storybook/builder-webpack5/templates/virtualModuleModernEntry.js.handlebars b/node_modules/@storybook/builder-webpack5/templates/virtualModuleModernEntry.js.handlebars
```

Comment:
Previously most of the packages were using webpack 4 while the Designer app was using webpack 5. I unified this and now everything is using webpack 5. However, I've encountered some hiccups with this setup and had to include 2 patches in this template~ to work around them. One of them is already available in the beta/next version of Storybook, but the other one isn't - it got rejected in favor of some other PR. However, that other PR is included in the version of the Storybook that you are using and it doesn't fix the issue that I bumped into. I expect that this issue might be fixed in the beta/next version of Storybook or that it will get fixed there in the future. I don't want to migrate to the "unstable" Storybook 7 yet - so I didn't recheck with that at the moment. 

To avoid using `patch-package` to work around those we could either:
- try to switch back to webpack 4 for all the existing storybooks
- keep using webpack 5 for the Designer app and keep using webpack 4 for packages

Let me know if using `patch-package` here is OK or if you prefer one of the other mentioned solutions.

Reply:
is webpack 5 giving us advantages there? I heard they are working on vite support ... personally I would go for simplicity - one webpack version for all

but that depends if its not super slow and what bug you had with wp5, can we live with the bug?

Reply:
> is webpack 5 giving us advantages there? 

Probably not. Just note that webdesigner app was already using webpack 5 so switching it to webpack 4 could also cost some work as things could break somehow unexpectedly. When unifying the webpack version I was facing a dilemma either way, so I just tried to use the newer version (webpack 5). Even though I had to hack something around - it didn't take too much time so I didn't have to cave and switch back to webpack 4 ;p

At this point, I don't see a strong reason to switch back to webpack 4 - as the setup here seems to be working. We can always revisit this too. I will also be reporting that one "unsolved" issue with webpack 5 to the Storybook team, so I hope that they will land a PR fixing it at some point.

> I heard they are working on vite support

They have it in their org but outside of their main monorepo: https://github.com/storybookjs/builder-vite . I could try to switch to that in a follow-up PR if you'd like that.

> but that depends if its not super slow and what bug you had with wp5, can we live with the bug?

I patched the bug - so I think that you can live with it for now. it's just that the patch that I used to work around it isn't yet a part of the "next" Storybook. Their maintainers already expressed an interest in taking a look at my repro case though - so there is a hope that they will fix this in their "core" sometime in the future. Note that those patches wouldn't immediately benefit you though. From what I understand they are working on their v7 and the current stable v6 doesn't receive a lot of updates.

Reply:
would be awesome if that works